zipline/pipeline/filters/filter.py

```diff
@@ -593,6 +595,10 @@ class MaximumFilter(Filter, StandardOutputs):
     window_length = 0
 
     def __new__(cls, factor, groupby, mask):
+        from zipline.pipeline.classifiers import Everything
```

Comment:
We can't import classifiers at module scope here because the `classifiers` module imports `filters`, because a bunch of the Classifier methods return Filter instances. I don't think there's a straightforward way to break that circular dependency more cleanly than this.